conans/client/downloaders/caching_file_downloader.py

```diff
@@ -158,10 +158,10 @@ def __init__(self, requester, config, scope=None):
             raise ConanException("core.download:download_cache must be an absolute path")
         self._file_downloader = FileDownloader(requester, scope=scope)
 
-    def download(self, url, file_path, auth, verify_ssl, retry, retry_wait):
-        if not self._download_cache:
+    def download(self, url, file_path, auth, verify_ssl, retry, retry_wait, metadata=False):
+        if not self._download_cache or metadata:  # Metadata not cached and can be overwritten
```

Comment:
The metadata cannot be cached, as it is not immutable, and not controlled by checksum